src/app/clusters/mode-base-server/mode-base-server.h

```diff
@@ -0,0 +1,198 @@
+/*
+ *
+ *    Copyright (c) 2023 Project CHIP Authors
+ *    All rights reserved.
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+#pragma once
+
+#include "mode-base-cluster-objects.h"
+#include <app/AttributeAccessInterface.h>
+#include <app/CommandHandlerInterface.h>
+#include <app/util/af.h>
+#include <map>
+
+using chip::Protocols::InteractionModel::Status;
+using ModeOptionStructType = chip::app::Clusters::detail::Structs::ModeOptionStruct::Type;
+using ModeTagStructType    = chip::app::Clusters::detail::Structs::ModeTagStruct::Type;
```

Comment:
No `using` in headers allowed. Move those to the .cpp, and in the header use the fully qualified names..

Reply:
Done [here](https://github.com/project-chip/connectedhomeip/pull/27504/commits/cbc7401ca9e94c6e835bed84cf97151ecf010feb).